concepticondata/conceptlists/Hsiu-2015-315.tsv

```diff
@@ -0,0 +1,317 @@
+Hsiu-2015-315-179	179	morning	1339	MORNING
+Hsiu-2015-315-180	180	mortar	224	MORTAR CRUSHER
+Hsiu-2015-315-181	181	mosquito	1509	MOSQUITO
+Hsiu-2015-315-182	182	mountain	639	MOUNTAIN
+Hsiu-2015-315-183	183	mouth	674	MOUTH
+Hsiu-2015-315-184	184	mud	640	MUD
+Hsiu-2015-315-185	185	mushroom	641	MUSHROOM
+Hsiu-2015-315-186	186	name	1405	NAME
+Hsiu-2015-315-187	187	narrow	1267	NARROW
+Hsiu-2015-315-188	188	navel	1838	NAVEL
+Hsiu-2015-315-189	189	near	1942	NEAR
+Hsiu-2015-315-190	190	neck	1333	NECK
+Hsiu-2015-315-191	191	needle	1382	NEEDLE (FOR SEWING)
+Hsiu-2015-315-192	192	new	1231	NEW
+Hsiu-2015-315-193	193	night	1233	NIGHT
+Hsiu-2015-315-194	194	nose	1221	NOSE
+Hsiu-2015-315-195	195	oil	1484	OIL (ORGANIC SUBSTANCE)
+Hsiu-2015-315-196	196	old (cu)	2113	OLD (USED)
+Hsiu-2015-315-197	197	old (gia)	2112	OLD (AGED)
+Hsiu-2015-315-198	198	open, to	1156	OPEN
+Hsiu-2015-315-199	199	oven	1143	OVEN
+Hsiu-2015-315-200	200	pig	1337	PIG
+Hsiu-2015-315-201	201	plant, to	1486	PLANT (SOMETHING)
+Hsiu-2015-315-202	202	plate	480	PLATE
+Hsiu-2015-315-203	203	plow	2154	PLOUGH (INSTRUMENT)
+Hsiu-2015-315-204	204	poor	1674	POOR
+Hsiu-2015-315-205	205	potato	593	POTATO
+Hsiu-2015-315-206	206	pound	354	POUND
+Hsiu-2015-315-207	207	rain	658	RAIN (PRECIPITATION)
+Hsiu-2015-315-208	208	rainbow	1733	RAINBOW
+Hsiu-2015-315-209	209	rat	1490	RAT
+Hsiu-2015-315-210	210	red	156	RED
+Hsiu-2015-315-211	211	rice, cooked	2750	GLUTINOUS RICE
+Hsiu-2015-315-212	211	rice, cooked	1191	COOKED RICE
+Hsiu-2015-315-213	212	rice, grain		
+Hsiu-2015-315-214	213	rice, plant	2026	RICE PLANT
+Hsiu-2015-315-215	214	rich	712	RICH
+Hsiu-2015-315-216	215	right	1725	CORRECT (RIGHT)
+Hsiu-2015-315-217	216	right side	1019	RIGHT
+Hsiu-2015-315-218	217	ripe	178	RIPE
+Hsiu-2015-315-219	218	road	667	ROAD
+Hsiu-2015-315-220	219	roof	769	ROOF
+Hsiu-2015-315-221	220	rooster	1511	ROOSTER
+Hsiu-2015-315-222	221	root	670	ROOT
+Hsiu-2015-315-223	222	rope	1218	ROPE
+Hsiu-2015-315-224	223	rotten	1728	ROTTEN
+Hsiu-2015-315-225	224	sad	699	SAD
+Hsiu-2015-315-226	225	salt	1274	SALT
+Hsiu-2015-315-227	226	salty	1091	SALTY
+Hsiu-2015-315-228	227	sand	671	SAND
+Hsiu-2015-315-229	228	second (#2)	1384	SECOND
+Hsiu-2015-315-230	229	see	1409	SEE
+Hsiu-2015-315-231	230	sell	1571	SELL
+Hsiu-2015-315-232	231	sew	1457	SEW
+Hsiu-2015-315-233	232	sharp	1396	SHARP
+Hsiu-2015-315-234	233	sheep	1331	SHEEP
+Hsiu-2015-315-235	234	shirt	1622	SHIRT
+Hsiu-2015-315-236	235	show	1039	SHOW
+Hsiu-2015-315-237	236	shoot	1172	SHOOT
+Hsiu-2015-315-238	237	short	1645	SHORT
+Hsiu-2015-315-239	238	shoulder	1482	SHOULDER
+Hsiu-2015-315-240	239	shrimp	2349	SHRIMP
+Hsiu-2015-315-241	240	sick	1847	SICK
+Hsiu-2015-315-242	241	silver	759	SILVER
+Hsiu-2015-315-243	242	sit	1416	SIT
+Hsiu-2015-315-244	243	skin	763	SKIN
+Hsiu-2015-315-245	244	skirt	1600	SKIRT
+Hsiu-2015-315-246	245	sky	1732	SKY
+Hsiu-2015-315-247	246	sleep	1585	SLEEP
+Hsiu-2015-315-248	247	small	1246	SMALL
+Hsiu-2015-315-249	248	smoke	2859	SMOKE (EMIT SMOKE)
```

Comment:
smoke is probably also just smoke "exhaust" as in concepticon. Did you check against HM languages in Chen?